operators/from_file.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package operators
+
+import (
+	"os"
+	"path"
+)
+
+func loadFromFile(filepath string, paths []string) ([]byte, error) {
+	if path.IsAbs(filepath) {
+		return os.ReadFile(filepath)
+	}
+
+	// handling files by operators is hard because we must know the paths where we can
+	// search, for example, the policy path or the binary path...
+	// CRS stores the .data files in the same directory as the directives
+	var (
```

Comment:
Not sure if this is needed

Reply:
The var or the comment? if it is the var I used it to declare error on beforehand to be able to return it in the last part.

Reply:
Oh I didn't notice the `err` being returned. It looks like if there are no paths, `err` won't be returned, so maybe you should return `os.ErrNotExist` explicitly there? Either way it could be clearer, predeclaring here seems to be making it hard to follow what makes it into the variable by the end of the function